generators/csharp/sdk/src/endpoint/request/WrappedEndpointRequest.ts

```diff
@@ -107,12 +108,14 @@ export class WrappedEndpointRequest extends EndpointRequest {
     }
 
     public getHeaderParameterCodeBlock(): HeaderParameterCodeBlock | undefined {
-        if (this.endpoint.headers.length === 0) {
+        const service = this.context.getHttpServiceOrThrow(this.serviceId);
+        const headers = [...this.endpoint.headers, ...service.headers];
```

Comment:
nit: I would put the service headers _before_ the endpoint headers (so that it's from _least_ specific to _most_ specific.

Reply:
i like that 👍